libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -215,11 +215,35 @@ bool SRV_Channel::is_motor(SRV_Channel::Aux_servo_function_t function)
 // return true if function is for anything that should be stopped in a e-stop situation, ie is dangerous
 bool SRV_Channel::should_e_stop(SRV_Channel::Aux_servo_function_t function)
 {
-    return ((function >= SRV_Channel::k_heli_rsc && function <= SRV_Channel::k_motor8) ||
-            function == SRV_Channel::k_starter || function == SRV_Channel::k_throttle ||
-            function == SRV_Channel::k_throttleLeft || function == SRV_Channel::k_throttleRight ||
-            (function >= SRV_Channel::k_boost_throttle && function <= SRV_Channel::k_motor12) ||
-            function == k_engine_run_enable);
+    switch (function)
+    {
+    case SRV_Channel::Aux_servo_function_t::k_heli_rsc:  
+    case SRV_Channel::Aux_servo_function_t::k_heli_tail_rsc:
+    case SRV_Channel::Aux_servo_function_t::k_motor1:
+    case SRV_Channel::Aux_servo_function_t::k_motor2:
+    case SRV_Channel::Aux_servo_function_t::k_motor3:
+    case SRV_Channel::Aux_servo_function_t::k_motor4:
+    case SRV_Channel::Aux_servo_function_t::k_motor5:
+    case SRV_Channel::Aux_servo_function_t::k_motor6:
+    case SRV_Channel::Aux_servo_function_t::k_motor7:
+    case SRV_Channel::Aux_servo_function_t::k_motor8:
+    case SRV_Channel::Aux_servo_function_t::k_starter:
+    case SRV_Channel::Aux_servo_function_t::k_throttle:
+    case SRV_Channel::Aux_servo_function_t::k_throttleLeft:
+    case SRV_Channel::Aux_servo_function_t::k_throttleRight:
+    case SRV_Channel::Aux_servo_function_t::k_boost_throttle:
+    case SRV_Channel::Aux_servo_function_t::k_motor9:
+    case SRV_Channel::Aux_servo_function_t::k_motor10:
+    case SRV_Channel::Aux_servo_function_t::k_motor11:
+    case SRV_Channel::Aux_servo_function_t::k_motor12:
+    case SRV_Channel::Aux_servo_function_t::k_engine_run_enable:
+        return true;
+
+    default:
```

Comment:
When I try to remove the default case, I am getting unhandled enumeration error. Any suggestions how to get rid of it?

Reply:
You would need to add a case for every Aux_servo_function_t.  Given that this is for estop I think it is worth the effort/cost to have them explicitly defined, hence it throws a wanring/error whenever someone adds to the function enum.

Reply:
@Gone4Dirt Won't it be better if I just add a default case instead. Because earlier too we were returning true explicitly for the concerned functions and false for the rest.

Reply:
@peterbarker Should I go for listing all the cases? It would cost us some space though.

Reply:
> @peterbarker Should I go for listing all the cases? It would cost us some space though.

Nah, not on this PR.

@Gone4Dirt raises the point that this *is* estop so enumerating them all is probably worth-while.  But this is an improvement and we shouldn't let perfect be the enemy of doesn't-suck.

I'll mark this for DevCall so we can decide if we burn the extra space on full enumeration.